tests/test_thumbnail.py

```
@@ -31,0 +1,250 @@
+#!/usr/bin/env python3
+"""
+Tests for the thumbnail generation functionality in thumbnail.py.
+"""
+
+import unittest
+from unittest.mock import patch, MagicMock, mock_open
```

Ruff (0.11.9)

`unittest.mock.mock_open` imported but unused

Remove unused import: `unittest.mock.mock_open`

(F401)